src/documentation/ToolkitDocumentation/ToolkitDocumentation.tsx

```diff
@@ -46,7 +53,7 @@ export const ToolkitDocumentation = ({ toolkit }: ToolkitProps) => {
   );
   return (
     <ActiveTooklitLevel
-      key={urlParam}
+      key={urlParam.length > 0 ? 0 : 1}
```

Comment:
Might be clearer for this to just use `anchor` now.

Reply:
Rewriting in terms of anchor here (and surrounding code here and in reference) improved things.